sessionctx/variable/session.go

```diff
@@ -765,8 +774,46 @@ func (s *SessionVars) WithdrawAllPreparedStmt() {
 	metrics.PreparedStmtGauge.Set(float64(afterMinus))
 }
 
+func (s *SessionVars) trySetWarningCountOrErrorCount(name string, val string) (bool, error) {
```

Comment:
We don't need to do this, `error_count` and `warning_count` is read-only

Reply:
Thank you for letting me know that.